### ISSUE TITLE
Weapons not changing correctly in Battledome : Fight!

### ISSUE THREAD

**Issue by birdgofly** [w35l3y/userscripts#58]
**Browser:** Firefox <br />
**Plugin:** Tampermonkey 4.5.5660<br />
**Script:** Battledome : Fight! 2.3.1<br />

When battling the wind rider wraith, I want to use the Winged Palm Blade, Downsize!, and the ability Meditate for the first round. For the second round, I want to use the Parchment Cloak, Cursed Elixer, and the ability An Icicle. 

The first round works great, but in the second round, the script will not switch out the Winged Palm Blade for the Cursed Elixer, so that the second round ends up being Parchment Cloak, Winged Palm Blade, and An Icicle. It doesn't seem to matter which slots I put which weapons into during setup, it always keeps the Winged Palm Blade and leaves out the Cursed Elixer. I've tried resetting the opponent stuff and reinstalling the script a number of times. 

Any idea what's going on?

Thank you!

**Comment by w35l3y** [comment]
That is right! Basically the script was designed to put items/abilities in open spaces. If a one-use item vanishes or breaks a multi-use item, then the script will place the next item in the order.

It probably is a duplicate of #2
